### DISCUSSION TITLE
My request is still pending

### DISCUSSION THREAD

**Discussion by YasinAllahverdi** [community/community#19547] [GitHub Education]
It said that I should receive an email from us by Jun 26. But it's Jun 27 today and I didn't receive an email.
My email is ******

**Comment by melvincwng** [comment]
@YasinAllahverdi, I presume you are requesting for the GitHub Student Education Pack?

If yes, please refer to #19545 for more information. I believe another user is also facing the same issue with regards to his GitHub Student Education Pack application status.

In short, I would suggest you raise a ticket to GitHub Education Support, and provide some additional time for GitHub to revert back to you on your application status.

Hope this clarifies.

**Reply by melvincwng** [reply]
@YasinAllahverdi, kindly follow the steps above and let me know if you face any issues.

Thank you and have a great day ahead 😌

**Reply by YasinAllahverdi** [reply]
![Screenshot_2022-06-27-16-18-12-719_com android chrome](https://user-images.githubusercontent.com/65732395/175934458-35769598-4ff8-4f3d-9dd7-a04c61f476f2.jpg)
 This is my screenshot of my request 
My username is @YasinAllahverdi

**Reply by melvincwng** [reply]
@YasinAllahverdi I see you have submitted on 24th June (Friday).

Given the current date of 27th June (Monday), that gives us a 3 day time difference. 

Out of those 3 days are two non-working days (Sat & Sun).

My suggestion would be for you to kindly wait for GitHub staff to review & approve your application for the Github Education Student pack.

Give GitHub some time to review your application. Meanwhile, please wait patiently for the time being.

If all fails, then you might want to consider raising a support ticket to them over [here](https://support.github.com/contact/education) for them to check why the application is still pending. 

Hope this clarifies.

**Reply by melvincwng** [reply]
@YasinAllahverdi, made some minor edits to my response above for better clarity.

 If this reply has been helpful, appreciate it if you could 'Mark as answer'. 

Also, feel free to follow me on GitHub as well, as I'm trying to expand my professional network of student developers (am a fellow student myself).

Thanks so much 🙏

**Comment by samus-aran** [comment]
Hi @YasinAllahverdi ,

I've removed your private information from [this post](https://github.com/community/community/discussions/19547) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).
